__init__.py

```
@@ -251,36 +254,76 @@ def stop_session(self, sess: Session) -> bool:
+
+
+WIKIHOW_PERSONA = {
+  "name": "Wikihow",
+  "solvers": [
+    "ovos-solver-wikihow-plugin",
+    "ovos-solver-failure-plugin"
+  ]
+}
+
 if __name__ == "__main__":
     from ovos_utils.fakebus import FakeBus
```

Ruff (0.8.2)

`ovos_utils.fakebus.FakeBus` imported but unused

(F401)